aca_preview/tests/test_checks.py

```diff
@@ -0,0 +1,21 @@
+# coding: utf-8
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+
+from proseco import get_aca_catalog
+from proseco.core import StarsTable
+from proseco.tests.test_common import DARK40, STD_INFO
+from aca_preview.preview import ACAReviewTable
+
+
+def test_check_P2():
+    """Test the check of acq P2"""
+    stars = StarsTable.empty()
+    stars.add_fake_constellation(n_stars=3, mag=10.25)
+    aca = get_aca_catalog(**STD_INFO, stars=stars, dark=DARK40)
+    ACAReviewTable.add_review_methods(aca)
```

Comment:
OK, so since the aca is somewhat converted to the ACAReviewTable, it is never pickled so the stars stay in the structure and set_stars doesn't fetch from the agasc?  Just trying to follow this through.

Reply:
Here `aca` is fully converted to `ACAReviewTable` (albeit by a tricky pathway).  There is no pickling and the `set_stars()` method is never called.  That only happens in the main program because in that case the original `ACATable` is from a pickle and doesn't already have stars.